config/ytt/overlay-manager-image.yaml

```diff
@@ -0,0 +1,13 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.subset({"kind": "Deployment"}), expects="1+"
```

Comment:
Is it worth maybe being specific about which Deployment is in the subset? Just in case one used this with a large group of manifests containing other Deployments.

Reply:
Good catch! I just hope `kustomize` doesn't mangle the name of the Deployment ever.